### PR TITLE
ASoC: Intel: ADL: Enable HDMI-In capture feature support for non-I2S …

### PR DESCRIPTION
**Pull request by CBala21**
…codec boards.

Adding HDMI-In capture support for the products doesn't have onboard I2S codec. but need to support HDMI-In capture via I2S and audio playback through HDMI/DP monitor.

### CODE REVIEW

**Comment by sofci** [comment]
Can one of the admins verify this patch?
> reply **test&nbsp;this&nbsp;please** to run this test once

File: sound/soc/intel/common/soc-acpi-intel-adl-match.c [hunk 1]
```diff
@@ -557,12 +557,6 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_adl_machines[] = {
 		.quirk_data = &adl_max98360a_amp,
 		.sof_tplg_filename = "sof-adl-max98360a-cs42l42.tplg",
 	},
-	/* place amp-only boards in the end of table */
-	{
-		.id = "CSC3541",
-		.drv_name = "adl_cs35l41",
-		.sof_tplg_filename = "sof-adl-cs35l41.tplg",
-	},
```
Comment (plbossart) [inline sound/soc/intel/common/soc-acpi-intel-adl-match.c:565]: moving this code block should be a separate patch.

**Review by plbossart — CHANGES_REQUESTED** [review]

File: sound/soc/intel/common/soc-acpi-intel-adl-match.c (hunk 1 above)
Comment (CBala21) [reply sound/soc/intel/common/soc-acpi-intel-adl-match.c:565]: Thanks for your review. Moving part has been removed from this patch and will upload as separate commit.

**Review by plbossart — DISMISSED** [review]

**Comment by plbossart** [comment]
test this please

**Review by plbossart — DISMISSED** [review]

File: sound/soc/intel/boards/sof_ssp_amp.c [hunk 2]
```diff
@@ -478,7 +487,7 @@ static struct platform_driver sof_ssp_amp_driver = {
 module_platform_driver(sof_ssp_amp_driver);
 
 MODULE_DESCRIPTION("ASoC Intel(R) SOF Amplifier Machine driver");
-MODULE_AUTHOR("balamurugan.c <balamurugan.c@intel.com>");
+MODULE_AUTHOR("Balamurugan C <balamurugan.c@intel.com>");
```
Comment (bardliao) [inline sound/soc/intel/boards/sof_ssp_amp.c:490]: This may need to be separate commit.

Comment (CBala21) [reply sound/soc/intel/boards/sof_ssp_amp.c:490]: Updated into separate commit

Comment (CBala21) [reply sound/soc/intel/boards/sof_ssp_amp.c:490]: Updated. please check

**Review by bardliao — DISMISSED** [review]

**Comment by bardliao** [comment]
test this please

**Comment by CBala21** [comment]
test this please

**Review by plbossart — APPROVED** [review]

**Comment by greg-intel** [comment]
SOFCI TEST

**Review by bardliao — APPROVED** [review]
